azure-devops/azext_devops/dev/pipelines/pipeline_create.py

```diff
@@ -155,12 +155,10 @@ def pipeline_create(name, description=None, repository=None, branch=None, yml_pa
                               project=project)
 
 
-def pipeline_update(name=None, id=None, description=None, new_name=None,  # pylint: disable=redefined-builtin
+def pipeline_update(id=None, description=None, new_name=None,  # pylint: disable=redefined-builtin
```

Comment:
we should remove `id=None` and only keep `id` 
because now id is mandatory

then this check can also be removed
```python
 if id is None:
        raise CLIError("--id argument must be supplied for this command.")
```

Reply:
Got it